### ISSUE TITLE
Document with incorrect viewing.

### ISSUE THREAD

**Issue by Akrobs** [ONLYOFFICE/DocumentServer#1639]
Do you want to request a feature or report a bug?
Bug

What is the current behavior?
Please, see attachments.

Did this work in previous versions of DocumentServer?
Yes

DocumentServer version:
7.0.0

Operating System:
CentOS 8.5

Browser version:
All

MS Office 2013
![Grafik_MS](https://user-images.githubusercontent.com/57731196/153898180-719a8797-d0b8-4e1a-b57b-d0dd823b8fd3.jpg)

OO
![Grafik_OO](https://user-images.githubusercontent.com/57731196/153898189-995aa85f-f632-4fa6-b192-c394bc0b374e.jpg)

**Comment by Akrobs** [comment]
I'm send file via email, witj issue number.

**Comment by ShockwaveNN** [comment]
Thanks, we recieved your file via private email, we'll take a look at this problem

**Comment by alinasafonova** [comment]
Duplicate of #1482

**Comment by ShockwaveNN** [comment]
I'll close it but feel free to comment if you feel that it's not a duplicated issue

**Comment by ShockwaveNN** [comment]
Yeah, but currently we have no solution for this problem so can't promise update any time soon
